appcli/templates/installer.j2

```diff
@@ -31,6 +53,12 @@ function main()
     rm -f "${_launcher_generic}" || print_error_and_exit "No permissions to delete to [${_launcher_generic}]"
     ln -s "${_launcher_versioned}" "${_launcher_generic}"
 
+    # Auto configure if enabled.
+    if [ $AUTO_CONFIGURE = true ]; then
+        {{ install_dir }}/{{ cli_context.app_name|lower }} configure init
```

Comment:
Should reference the already-existing `${_launcher_generic}`

Reply:
Also use double square brackets for tests `[[`, its a better alternative to single square brackets `[`